src/preconnect.js

```diff
@@ -135,6 +145,20 @@ class Preconnect {
     return false;
   }
 
+  /** @private */
+  isPreloadSupported_() {
```

Comment:
Memoizing would really help here, avoiding a bunch of useless `<link>`s from being created.